.pre_commit/generate-facets/templates/dataclass.jinja2

```diff
@@ -0,0 +1,116 @@
+{%- import 'validators.jinja2' as validators -%}
+
+{%- set private_prefix = '' -%}
+{%- if class_name.endswith('Facet') -%}
+{%- set private_prefix = '_' -%}
+{%- endif -%}
+{%- if class_name != "ClassToBeSkipped" -%}
+{% for decorator in decorators -%}
+{{ decorator }}
+{% endfor -%}
+@define
+{%- if class_name.endswith("Event") -%}(kw_only=True){%- endif -%}
+{%- if base_class %}
+class {{ class_name }}({{ base_class }}):
+{%- else %}
+class {{ class_name }}:
+{%- endif %}
+{%- if description %}
+    """
+    {{ description | wordwrap(100) | indent(4)}}
+    """
+{%- endif %}
+{%- if not fields and not description %}
+    pass
+{%- endif %}
+{%- for field in fields if field.name not in [private_prefix+'schemaURL', private_prefix+'producer'] -%}
+    {%- if 'uuid' in field.type_hint -%}
+        {%- set type_hint = field.type_hint.replace('uuid', 'str') -%}
+    {%- elif 'date-time' in field.type_hint -%}
+        {%- set type_hint = field.type_hint.replace('date-time', 'str') -%}
+    {%- else -%}
+        {%- set type_hint = field.type_hint -%}
+    {%- endif -%}
+    {%- if field.field %}
+    {{ field.name }}: {{ type_hint }} = 
```

Comment:
Can this be outside of if ? The one in else is simply without the `=`

Reply:
In `else` block there's a case when there doesn't have to be anything after type hint. This happens when:
* there's no default value
* no `kw_only=True` argument needed
* it's not validated
* `field(factory=list/dict)` for lists/dicts in type hints